.github/workflows/deploy_testnet.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Testnet
+
+on:
+  push:
+    branches:
+    - '*'
```

Comment:
Haha the collisions this produces is going to be interesting. Perhaps we should just target `master`, or an internal development branch?

I was thinking, if we migrate this subgraph into the `aragon-court` repo, we could keep a `graph-staging` branch that pushes to a separate staging subgraph URL.